templates/test-environment-template.js

```diff
@@ -1,12 +1,12 @@
 function makeTestEnvironment(_, simple) {
-  var requireAccess = simple.stub();
```

Comment:
Since the sync function template's `.jshintrc` was moved into that subdirectory, this file is free to use ECMAScript 2015 features like `const`.